snapcraft/parts/lifecycle.py

```diff
@@ -271,6 +271,8 @@ def _run_command(
         work_dir=work_dir,
         assets_dir=assets_dir,
         base=project.get_effective_base(),
+        project_base=project.base or "",
```

Comment:
This is needed because craft-parts expects `base` to be the effective base (i.e. build-base if base is bare).